docs/private-networks/reference/cli/options.md

```diff
@@ -922,6 +922,48 @@ privacy-url="http://127.0.0.1:8888"
 
 The URL on which the [Tessera node](../../tutorials/privacy/index.md#3-create-tessera-configuration-files) is running.
 
+### `snapsync-bft-enabled`
+
+<Tabs>
+
+<TabItem value="Syntax" label="Syntax" default>
+
+```bash
+--snapsync-bft-enabled[=<true|false>]
+```
+
+</TabItem>
+
+<TabItem value="Example" label="Example">
+
+```bash
+--snapsync-bft-enabled=true
+```
+
+</TabItem>
+
+<TabItem value="Environment variable" label="Environment variable">
+
+```bash
+BESU_SNAPSYNC_BFT_ENABLED=true
+```
+
+</TabItem>
+
+<TabItem value="Configuration file" label="Configuration file">
+
+```bash
+snapsync-bft-enabled=true
+```
+
+</TabItem>
+
+</Tabs>
+
+Enables or disables snapsync in IBFT 2.0 and QBFT private networks. The default is `false`.
+
+Use `--snapsync-bft-enabled` with `--sync-mode=SNAP` to enable snapsync in QBFT and IBFT 2.0 private networks.
+
```

Comment:
so if the option is being removed altogether, we DON'T want this change (addition)

Reply:
I agree, will cleanup